cvat-ui/src/components/cvat-app.tsx

```diff
@@ -358,6 +359,8 @@ class CVATApplication extends React.PureComponent<CVATAppProps & RouteComponentP
                                     </Switch>
                                 </GlobalHotKeys>
                                 {/* eslint-disable-next-line */}
+                                <ExportDatasetModal />
```

Comment:
I saw it was related to the original PR, but could you explain, why do we need to move modal to cvat-app? As I see there is no other modals here

Reply:
As far as I know after dismounting it still stays in DOM, and current antd version adds some css IDs (based on fields names).
So, when we go between pages we get duplicated ids and corresponding errors. 

I've added ``destroyOnClose``, and than decided that this component is quite often (projects page, project page, task page, tasks page, annotation view) and put it here.

Reply:
I guess, better to keep it consistent and keep modals where they need. It only question of readability and I'm not against your decision